elements/pfe-accordion/src/pfe-accordion-header.js

```diff
@@ -196,8 +196,10 @@ class PfeAccordionHeader extends PFElement {
         // Capture it's assigned nodes for validation
         let slotted = node.assignedNodes();
         // If slotted elements were found, add it to the nodeList
-        if (slotted && slotted.length > 0) nodes[idx] = slotted;
-        else {
+        if (slotted && slotted.length > 0) {
+          // Remove the slot from the set, add the slotted content
+          nodes.splice(idx, 1, ...slotted);
```

Comment:
This was causing the bug in content-set where accordion headers were all returning "undefined"